### PR TITLE
Fix/drone update

### PR DESCRIPTION
**Pull request by acien101**
Updating drone/gitea installation scripts.

Using this documentation (the newest one right now): 

https://web.archive.org/web/20191013214221/https://docs.drone.io/installation/providers/gitea/

https://web.archive.org/web/20191013214332/https://docker-runner.docs.drone.io/installation/install_linux/

### CODE REVIEW

File: full.yml [hunk 1]
```diff
@@ -555,17 +555,18 @@
...
       -e "DRONE_GITEA_SERVER=https://{{ gitea_domain }}" \
+      -e "DRONE_GITEA_CLIENT_ID={{ gitea_oauth_client_id }}" \
+      -e "DRONE_GITEA_CLIENT_SECRET={{ gitea_oauth_client_secret }}" \
+      -e "DRONE_RPC_SECRET={{ drone_rpc_scret }}" \
       -e "DRONE_GIT_ALWAYS_AUTH=false" \
       -e "DRONE_RUNNER_CAPACITY=2" \
       -e "DRONE_SERVER_HOST={{ drone_domain }}" \
       -e "DRONE_SERVER_PROTO=https" \
       -e "DRONE_TLS_AUTOCERT=false" \
-      --network "{{ gitea_network }}" \
```
Comment (m0wer) [inline full.yml]: ¿Why? I think we should leave this network and add it to drone runner also for an easier and more secure setup.

File: full.yml [hunk 2]
```diff
@@ -603,6 +604,30 @@
...
+      -e "DRONE_RPC_PROTO=https" \
+      -e "DRONE_RPC_HOST={{ drone_domain }}" \
+      -e "DRONE_RPC_SECRET={{ drone_rpc_scret }}" \
+      -e "DRONE_RUNNER_CAPACITY=2" \
+      -e "DRONE_RUNNER_NAME={{ drone_runner_name }}" \
+      "{{ docker_image }}"
+  roles:
+    - role: generic_docker_systemd
+  tags: drone
+
```
Comment (m0wer) [inline full.yml]: If docker runner is deployed in docker it won't be able to execute things outside of it right? For example `molecule test` using Vagrant.

File: full.yml (hunk 1 above)
Comment (acien101) [reply full.yml]: Its useless. With an external endpoint using oauth2 is not necessary to be on the same network. It not easier or secure due its useless.

File: full.yml (hunk 2 above)
Comment (acien101) [reply full.yml]: On the next PR I will add docker exec that can run molecule test. Sometimes its better to run pipelines with docker. The best idea is to have the docker runner and the exec runner.

Comment (m0wer) [reply full.yml]: Great :-)

**Comment by m0wer** [comment]
So just one more thing before we merge, add documentation for drone-runner and tests in molecule please.

**Comment by acien101** [comment]
Where do I have to put the documentation?

**Comment by m0wer** [comment]
> Where do I have to put the documentation?

In *docs/services/drone.md*.

**Comment by m0wer** [comment]
Thanks @acien101 ! Merging.

**Pull request merged**